VCAC-A/script/setup-hddl.sh

```diff
@@ -0,0 +1,139 @@
+#!/bin/bash
+
+DIR=$(dirname $(readlink -f "$0"))
+openvino_name="l_openvino_toolkit_p_2019.3.334"
+ov_link="http://registrationcenter-download.intel.com/akdlm/irc_nas/15944/$openvino_name.tgz"
+py_ver='3.7.2'
+py_link="https://www.python.org/ftp/python/$py_ver/Python-$py_ver.tgz"
+pack_dir="$( cd "$(dirname "$0")" && pwd )"
+package_name="intel-vcaa-hddl"
+ov_hddl="ov_hddl"
+ov_hddl_version="v1"
+hddldaemon_cfg="hddldaemon.cfg"
+
+build_hddl_package()
+{
+	gen_Dockerfile
+	docker build -t ${ov_hddl}:${ov_hddl_version} ${DIR} $(env | grep -E '(proxy|REPO|VER)=' | sed 's/^/--build-arg /')
+        docker run --rm $(env | grep -E '(proxy)=' | sed 's/^/-e /') ${ov_hddl}:${ov_hddl_version}
+        local container_id=`docker ps -a |grep ${ov_hddl}:${ov_hddl_version} | awk '{ print $1 }'`
+        for cont_id in $container_id;
+        do
+            container_id=$cont_id
+            break
+        done
+        docker cp $container_id:/root/package/${package_name}.tar.gz $pack_dir
+        docker cp  $container_id:/root/package/ov_ver.log $pack_dir
+}
+
+transfer_hddl_package()
+{
+
+	image="$1"
+	worker="root@$nodeip"
+ 
+	echo "Update image: $image to $worker"
+	echo "Transfering image..."
+	scp "$image" $NODEUSER@$nodeip:/root/${package_name}.tar.gz 
+}
+
+install_hddl_package()
+{
+	ov_ver=`cat ov_ver.log`
+	mkdir -p /opt/intel/
+	tar zxf ${package_name}.tar.gz -C /opt/intel/
+	mv /opt/intel/openvino /opt/intel/$ov_ver
+        ln -s /opt/intel/$ov_ver /opt/intel/openvino
+        export LC_ALL=C
+        apt-get install -y cmake libelf-dev
+	cd /opt/intel/$ov_ver/deployment_tools/inference_engine/external/hddl/drivers/drv_ion 
+	make && make install
+	if [ $? != 0 ];then
```

Comment:
`if ! { make && make install; }; then`

Reply:
$(env | grep -E '(proxy|REPO|VER)=' | sed 's/^/--build-arg /') should be added before ${DIR} in docker build.